src/difference.js

```diff
@@ -4,7 +4,7 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Finds the set (i.e. no duplicates) of all elements in the first list not
- * contained in the second list.
+ * contained in the second list (in terms of R.contains).
```

Comment:
I definitely prefer @buzzdecafe's formulation,  "Elements are compared in terms of *value* equality, not *reference* equality."  I feel as though the things we document are the things we're committing to.  And I feel little need to commit to what seems an implementation detail.

It's odd, because I probably would not object in certain places, such as `lift`, where the fact that it's implemented in terms of `ap` and `map` is to my mind quite important.  Even if we could write a more efficient implementation, I wouldn't be interested, because this this now works with all sorts of additional types for free.

So I guess I'm not entirely consistent on this, but I think my take makes reasonable sense.